tests/unit/test_report.py

```
@@ -0,0 +1,89 @@
+        runner = self._make_runner([])  # empty results list
+        # Remove 'results' from context to simulate a runner with no pre-loaded results
+        # (mirrors what report_show does when building runner without runner.results)
+        if 'results' in runner.context:
+            del runner.context['results']
+        report = Report(runner)
+        # Should not crash and data structure must be valid
+        report.build(query={})
+        assert 'results' in report.data
+        assert 'info' in report.data
+        # Verify context was not poisoned with an empty list
+        import secator.report as report_module
```

Flake8 (7.3.0)

[error] 'secator.report as report_module' imported but unused

(F401)